app/src/main/java/com/eis/phoneringer/structure/RingCommandHandler.java

```diff
@@ -0,0 +1,51 @@
+package com.eis.phoneringer.structure;
+
+import com.eis.smslibrary.SMSPeer;
+
+/**
+ * Class used to check if password in memory corresponds to the password passed through sender
```

Comment:
This class is actually used to parse commands to sms and back: update this description